packages/examples/packages/signature-insights/src/index.ts

```diff
@@ -0,0 +1,115 @@
+import type { OnSignatureHandler } from '@metamask/snaps-sdk';
+import { SeverityLevel, panel, text, row, heading } from '@metamask/snaps-sdk';
+
+/**
+ * The MetaMask test dapp uses the below contract address as the verifying
+ * contract for `eth_signTypedData_v3` and `eth_signTypedData_v4`, it is being
+ * used here as a flag to return content and to make e2e testing easier in the extension.
```

Comment:
Can you provide a permanent link to where this contract is set-up

Reply:
It's a dummy contract address, I guess I should have specified that